packages/css/search.css

```diff
@@ -57,31 +53,26 @@
     display: none;
   }
 
-  &:has(button[type='reset']) {
-    & input {
-      padding-inline-end: calc(var(--dsc-search-clear-button-size) + var(--dsc-search-padding-inline-x) * 2);
-    }
+  &:has(button[type='reset']) input {
+    padding-inline-end: calc(var(--dsc-search-clear-size) + var(--dsc-search-padding-inline));
   }
 
   & button[type='reset'] {
+    --dsc-button-size: var(--dsc-search-clear-size);
+
     grid-area: 1 / 1;
     justify-self: end;
-    align-self: center;
-    margin-inline-end: calc(var(--dsc-search-padding-inline-x) * 2);
-    height: var(--dsc-search-clear-button-size);
-    width: var(--dsc-search-clear-button-size);
-    font-size: 0.5em;
```

Comment:
The reason I added `font-size: 0.5em` was to make sure the button is smaller than the input.
It is currently too big, and our button does not come in a smaller size.
![image](https://github.com/user-attachments/assets/8bd1b51a-cdc8-4ba9-ad7a-e6d361341cea)

Reply:
Right ^^ I see the issue, but setting `font-size` also scales padding and everything - maybe this is okay since we know the input and button is same size ☺️ actually, come to think of it - maybe we should rather use `scale: .75` - then spacing to the edges will come automatically 🤩